test/evergreen.yml

```diff
@@ -1766,3 +1766,17 @@ buildvariants:
   tasks:
   - name: compile
   - name: unit-test
+
+- name: ubuntu1804-zseries
+  display_name: Ubuntu 18.04 ZSeries
+  run_on:
+  - ubuntu1804-zseries-test
+  expansions:
+    configure_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH
```

Comment:
Why we are not building with mongodb toolchain here

Reply:
I have updated the "configure_env_vars" to set PATH and CFLAGS.